### DISCUSSION TITLE
Play sound

### DISCUSSION THREAD

**Discussion by bluecells** [Budibase/budibase#8166] [Help]
Hi,

First post. Is there a way to play a sound file? The play function is "undefined" on the server side. So I have tried to create an automation with: 

if(typeof Audio != "undefined") {
	const audio = new Audio("https://www.fesliyanstudios.com/play-mp3/387");
	const buttons = document.querySelectorAll("button");
	buttons.forEach(button => {
	button.addEventListener("click", () => {
	audio.play();
	});
	});
};

Code results OK but no sound. I really need this sound feature.

Thanks a lot,
Jerome

**Comment by bluecells** [comment]
Little up. Is there a way to make node.js beep? Or do I need to install Budibase locally?
This audio feedback is very important for my app. Your insight is very much appreciated.

Thanks

**Comment by melohagan** [comment]
Hey @bluecells 

There is no play audio event, however you could potentially build this as a custom component: https://docs.budibase.com/docs/custom-plugin
